src/github.rs

```diff
@@ -767,9 +767,9 @@ impl Repository {
             .any(|&(key, value)| key == "is" && value == "pr");
         // negating filters can only be handled by the search api
         let url = if is_pr {
-            self.build_pulls_url(filters, include_labels)
+            self.build_issues_pulls_url(filters, include_labels)
         } else if use_issues {
-            self.build_issues_url(filters, include_labels)
+            self.build_issues_pulls_url(filters, include_labels)
```

Comment:
I'd like to remove this redundant if/else conditional in a follow-up patch after properly testing this PR